packages/w/wigxjpf/xmake.lua

```diff
@@ -0,0 +1,29 @@
+package("wigxjpf")
+    set_homepage("https://fy.chalmers.se/subatom/wigxjpf/")
+    set_description("WIGXJPF evaluates Wigner 3j, 6j and 9j symbols accurately using prime factorisation and multi-word integer arithmetic.")
+    set_license("GPL-3.0", "LGPL-3.0")
+
+    set_urls("https://fy.chalmers.se/subatom/wigxjpf/wigxjpf-$(version).tar.gz")
+    add_versions("1.13", "90ab9bfd495978ad1fdcbb436e274d6f4586184ae290b99920e5c978d64b3e6a")
+
+    on_install("linux", "macosx", function (package)
+        local configs = {}
+        import("package.tools.make").build(package, configs)

```

Comment:
We need to distinguish between static/shared libraries based on `package:config("shared")` if it only provides static libraries, please mark it as static readonly. 

```
add_configs("shared",      {description = "Build shared library.", default = false, type = "boolean", readonly = true})
```

Reply:
In the Makefile, the shared library is maked by a extra target, thus by running `make shared`. I add some code for the condition.